control/trajectory_follower_node/config/plot_juggler_trajectory_follower.xml

```diff
@@ -241,33 +241,42 @@
+   <scripts/>
+  </plugin>
+  <plugin ID="CSV Exporter"/>
+  <plugin ID="ROS2 Topic Re-Publisher"/>
  </Plugins>
  <!-- - - - - - - - - - - - - - - -->
  <previouslyLoaded_Datafiles/>
- <previouslyLoaded_Streamer name="ROS 2 Topic Subscriber"/>
+ <previouslyLoaded_Streamer name="ROS2 Topic Subscriber"/>
```

Comment:
It seems that we can no longer use "ROS 2 Topic Subscriber" as before.
![image](https://github.com/tier4/autoware-spell-check-dict/assets/59680180/db5d3804-9c1e-4faf-92c4-37cd464d1906)

Reply:
https://github.com/tier4/autoware-spell-check-dict/pull/720
I have created a pull request to autoware-spell-check-dict to allow the string 'ROS2'.

Reply:
[set spell-check ignore labels of for "ROS2"](https://github.com/autowarefoundation/autoware.universe/pull/6910/commits/456a83c5d507882234002b64c4f2e1d255f7d2d5)